common/utils.py

```diff
@@ -342,3 +344,48 @@ def init_logging(mode=None):
 	logging.config.fileConfig(logging_conf)
 	global log
 	log = logging.getLogger('utils')
+
+def get_user_id(name):
```

Comment:
Good point... they were used at first but then I ended up not using them. I left them in because I thought they might be useful, but I guess `twitch.get_user` ended up being just better in general...